### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN033.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#80] [General]
# https://blog.3place.cn/3place/lmyn/LMYN033.html

https://blog.3place.cn/3place/lmyn/LMYN033.html

**Comment by picklecai** [comment]
乾闼婆：在印度神话中，原来是一群半神半人的天上乐师，是帝释天属下职司雅乐的天神。 此神经常住在地上的宝山之中，有时升至忉利天演奏天乐，善于弹琴，演奏种种奇妙的雅乐。乾闼婆是东方持国天的眷属，为守护东方的神，有众多眷属。其同时也是观音二十八部众之一。 有经文记载其形象为，顶上有八角冠，身相为赤肉色，身如大牛王，或者其他动物的，如马或者鸟；左手执萧笛，右手执宝剑，具大威力相，发髻有焰鬘冠。

**Comment by picklecai** [comment]
国王仅仅把她们的手一握，
她们就不再驼背扫尽愁云，
这一百个国王的女儿们
一个个又焕发出美丽青春。 （1.32.23）

能治愈恋爱伤痛的，是新的恋情。
所以这位国王轻松扫除了风神留下的伤害。

**Comment by picklecai** [comment]
这个光辉的国王
就把梵授叫了来，
把一百女儿给了他，
心里面满怀愉快。 （1.32.21）

将一百个女儿都嫁给同一个人，居然还光辉。我为女儿哭。
最爽的爽文，也是让男主，人见人爱，一个一个地去爱他的。可不敢让100个人同一时间爱上了他。
